ui/pages/settings/networks-tab/networks-tab.component.js

```diff
@@ -56,12 +56,13 @@ export default class NetworksTab extends PureComponent {
         </span>
         <div className="networks-tab__add-network-header-button-wrapper">
           <Button
-            type="secondary"
+            type="primary"
             onClick={(event) => {
               event.preventDefault();
               setSelectedSettingsRpcUrl('');
               setNetworksTabAddMode(true);
             }}
+            rounded
```

Comment:
May be `rounded` can be made default if most buttons are rounded.

Reply:
I agree I think this is definitely worth considering! @danjm @brad-decker @rachelcope should we go ahead and make this change? or ticket it up?

Reply:
After discussion with @brad-decker we have decided to make the changes to remove the prop in this PR itself. I will be knocking on doors for more reviews very soon 😅